config/config.yml

```diff
@@ -276,3 +276,4 @@ service:
     level: INFO
     file-writer:
       filename: "log/backup-service.log"
+  date-format: EU
```

Comment:
It is better to create another subsection for backup configuration, e.g.:
```yaml

service:
  logger:
    level: INFO
    file-writer:
      filename: "log/backup-service.log"
  backup:
    timestamp-format: EU
```
If not specified or (NONE?) is chosen, no formatting is applied, keeping the existing timestamp.